README.md

```diff
@@ -181,6 +182,31 @@ Optional steps:
 [bash completions]: extra/completions/liquidctl.bash
 
 
+## Installing on FreeBSD
+
+liquidctl is maintained in the Ports Collection (thanks to ehaupt@FreeBSD.org), and it is available as a pre-built binary package.
+
+Port:
+
+`sysutils/py-liquidctl`
+
+Binary:
+
+`pkg install py37-liquidctl`
+
+Dependencies:
+
+`devel/py-docopt`
+
+`comms/py-hidapi`
+
+`devel/py-pyusb`
+
+liquidctl requires root privileges (doas or sudo) by default.
```

Comment:
```suggestion
By default, root privileges (`doas` or `sudo`) are required to run liquidctl.
```

---

Also, is it possible/advised to change this default behavior?

On Linux this is done with udev rules (see `extra/linux/`).

Reply:
I _think_ that would best be done with devd(8), of which the GNU userland's udev utility is a (distant) imitation/clone, but I will have to spend a lot of time digging through the man pages to figure out how to get it setup properly for myself before writing a guide -- or else someone who is used to working with such things will need to add that instead.

Until then, I suppose that using `chown` for the file of the USB device is not a downright terrible option, e.g. `sudo chown [user] /dev/ugen[#.#],` as long as it is followed by a careful `chmod 644` (or stricter). But...maybe we should not formally suggest such a bizarre thing. 

I am learning.

Reply:
It would be interesting to have this on the docs, but it doesn't need to get in right now.  Take your time...

On Linux we're currently using udev to set a `+uaccess` tag, which is pretty much equivalent to `chmod 0666`.  This is fairly common, even though I don't love it; I've recently pinged some package maintainers to discuss how to improve this.

My idea is something like setting the group to `wheel`, or some other common admin group; the issue is that Linux distros tend to have slightly different ways of handling system admin privileges, and some of them are more complicated than just a special group (and in those distros, these groups may not even exist by default).